packages/jira-adapter/src/filters/fields/field_type_references_filter.ts

```diff
@@ -32,45 +32,69 @@ export const getFieldsLookUpName: GetLookupNameFunc = ({
   return ref
 }
 
+const getDefaultOptions = (instance: InstanceElement) : {
+    referencedDefaultOptions: Values[]
+    referencedCascadingDefaultOption: Values | undefined } => {
+  let referencedDefaultOptions: Values[]
+  let referencedCascadingDefaultOption: Values | undefined
+
+  if (instance.value.defaultValue.type.endsWith('multiple')) {
+    referencedDefaultOptions = (Object.values(instance.value.options ?? {}) as Values[])
+      .filter((option: Values) => (instance.value.defaultValue.optionIds ?? []).includes(option.id))
+  } else {
+    referencedDefaultOptions = (Object.values(instance.value.options ?? {}) as Values[])
+      .filter((option: Values) => instance.value.defaultValue.optionId === option.id)
+    if (referencedDefaultOptions.length > 0) {
+      referencedCascadingDefaultOption = (
+        _.values(referencedDefaultOptions[0].cascadingOptions ?? {})
+      ).find((option: Values) => option.id === instance.value.defaultValue.cascadingOptionId)
+    }
+  }
+  return { referencedDefaultOptions,
+    referencedCascadingDefaultOption }
+}
```

Comment:
nit: to avoid `let` and indented code

```suggestion
  if (instance.value.defaultValue.type.endsWith('multiple')) {
    const referencedDefaultOptions = (Object.values(instance.value.options ?? {}) as Values[])
      .filter((option: Values) => (instance.value.defaultValue.optionIds ?? []).includes(option.id))
    return { referencedDefaultOptions }
  }
  const referencedDefaultOptions = (Object.values(instance.value.options ?? {}) as Values[])
    .filter((option: Values) => instance.value.defaultValue.optionId === option.id)

  const referencedCascadingDefaultOption = referencedDefaultOptions.length > 0
    ? (
      _.values(referencedDefaultOptions[0].cascadingOptions ?? {})
    ).find((option: Values) => option.id === instance.value.defaultValue.cascadingOptionId)
    : undefined
    
  return { referencedDefaultOptions, referencedCascadingDefaultOption }
}
```